react/features/base/config/functions.js

```diff
@@ -50,6 +50,7 @@ const WHITELISTED_KEYS = [
     'disableDesktopSharing',
     'disableH264',
     'disableHPF',
+    'displayName',
```

Comment:
1. @danielob, `displayName` sounds too generic to me in the context of config.js. I'll rename it to `callDisplayName` so that at least there's a parallel with `callUUID`.
2. In addition to `callUUID`, these properties need documenting comments. Let's in the future make sure that we add these.